README.md

```diff
@@ -41,6 +41,8 @@ Poll Settings provide further customisation, e.g. `/poll "Is Matterpoll great?"
 - `--anonymous`: Don't show who voted for what at the end
 - `--progress`: During the poll, show how many votes each answer option got and, in post card, show who voted for which answers ([#431](https://github.com/matterpoll/matterpoll/pull/431))
 - `--public-add-option`: Allow all users to add additional options
+- `--end=2006-01-02T15:04`: End poll automatically at 2006-01-02 at 15:04 or
```

Comment:
What about the time zone?  
Also mention ISO 8601

Reply:
I use the timezone of the author of the poll. I think it is more user-friendly

Reply:
Hmm.. Then this should be mentioned in the documentaion

Reply:
I agree that it's better to clarify which timezone will be used, and not to handle seconds. (And since [the code parsing datetime string](https://github.com/matterpoll/matterpoll/pull/544/files/6c5b8c5ccf2a2376b48b907c0d072ffda69ee1bf#diff-320751a0c35c5cde04e00dda2b10c4098126fd0486ca09ca85131bfe09607814R183) will be running on Mattermost instance, I think that the timezone of the instance is used, not of the author. Is my understanding correct?)

Reply:
I am almost back from holiday :blush: @kaakaa You are write, the timezone of the instance is used.

- I updated the help text
- Added more ways to parse the date (pass with or without seconds and timezone)
- Moved some logic from poll.go to new setting.go file

I think I did everything I wanted and PR is ready for review